### DISCUSSION TITLE
How to deactivate or change banner in homepage?

### DISCUSSION THREAD

**Discussion by GPlay97** [evershopcommerce/evershop#501] [Q&A]
Documentation only covers some components, but the "MainBanner", which is shown within homepage, a discount information, seems to not be editable, because it is not a component, I guess, but a module? How to handle that? I want to change background, text and so.. Ok.. for background at least, i can modify css. But there must be an easy way, or not?

**Comment by leandrosantoss** [comment]
@treoden 

The documentation talks about changing the banner. But how do I change the writing too? I don't want to talk about discounts, coupons, etc. Maybe just the image.

![image](https://github.com/evershopcommerce/evershop/assets/59751373/1a88ea06-2c08-44eb-b013-17af4df73dbf)

Code:
![image](https://github.com/evershopcommerce/evershop/assets/59751373/6cc3351b-b17a-4693-9243-b21efe463863)

Thanks

**Reply by leandrosantoss** [reply]
Would the solution be to use evetheme?

**Reply by treoden** [reply]
Hi @leandrosantoss ,
I assume you have your own theme, it can be the evetheme or your own custom theme.

So now you want to have your own banner. Let's create a component from inside your theme following this path

`<Your theme>/pages/homepage/MainBanner.jsx `

From here you can write your own component display whatever you need. EverShop will pickup this component and replace the core for you.

Thanks

**Reply by leandrosantoss** [reply]
Perfect. Thanks
